packages/examples/package.json

```diff
@@ -23,7 +23,8 @@
     "@nova/types": "1.5.1",
     "graphql": "^15.5.0",
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "react-relay": "^17.0.0"
```

Comment:
This is probably better as a peer dependency?

I hope to upgrade to relay 18 soon, which would mean this won't "align" anymore.

Reply:
Oh this is the examples package nvm :D